types/activity/submission.ts

```diff
@@ -17,8 +18,8 @@ export type SubmissionCreatedActivityItem = SubmissionActivityItemBase & {
 	kind: 'submission-created';
 };
 
-export type SubmissionStatusChangedActivityItem = SubmissionActivityItemBase & {
-	kind: 'submission-status-changed';
+export type SubmissionStatusUpdatedActivityItem = SubmissionActivityItemBase & {
```

Comment:
I think we match the type name and the `kind` value everywhere. If so, either `submission-status-updated` as the `kind` or `SubmissionUpdatedActivityItem` as the type name would true them up